README.adoc

```diff
@@ -8,26 +8,22 @@
+*The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-cmake-llvm-cov-gh-actions-sc[SonarCloud]!*
 
 :note-caption: :information_source:
-NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
-
-It is very easy to analyze a C, C++ and Objective-C project with SonarCloud with coverage and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
-
-. Create a `sonar-project.properties` file to store your configuration
-. In your `.github/workflows/build.yml` file:
-.. Download the https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/[Sonar Scanner]
-.. Download the https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-build-wrapper[Build Wrapper]
-.. Wrap your compilation with the Build Wrapper
-* Add the `-fprofile-instr-generate -fcoverage-mapping` compilation flags to instrument the binaries
-.. Run the tests to generate the test coverage information
-* After each test run, "default.profraw" file will be generated. Make sure to move it someplace, to avoid overwriting with another test run.
-.. Merge the coverage information from all test runs and convert the result into a format compatible with llvm-cov using `llvm-profdata merge`
-.. Run `llvm-cov show` on all the test executable files (or all object files you are interested in).
-* Save the output (the source files annotated with coverage counters) into a file
-.. Run `sonar-scanner` as the final step with `sonar.cfamily.llvm-cov.reportPath` pointing to the file with coverage report
-. Make sure that you have your token stored as a secret in your repository (`SONAR_TOKEN`). You can request new tokens using https://sonarcloud.io/account/security/[Account/Security page].
+NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?language=&page=1&q=topic%3Aconfiguration&sort=&type=all[configuration examples] are available here. +
+Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.
+
+To get coverage and feed it to the sonar-scanner, use the following procedure.
+
+. Add the `-fprofile-instr-generate -fcoverage-mapping` compilation flags to instrument the binaries
+. Run the tests to generate the test coverage information
+.. After each test run, "default.profraw" file will be generated. Make sure to move it someplace, to avoid overwriting with another test run.
```

Comment:
```suggestion
.* After each test run, "default.profraw" file will be generated. Make sure to move it someplace, to avoid overwriting with another test run.
```